docs/jenkins/README.md

```diff
@@ -0,0 +1,109 @@
+# In-house cluster setup
+
+```bash
+su
+sh install.sh
+docker-compose up -d
+```
+
+To rebuild the stack run:
+
+```bash
+docker-compose down
+docker-compose build
+docker-compose up --force-recreate -d
```

Comment:
Can't we use?
`docker-compose up --force-recreate --build -d`

Reply:
Indeed we should be able to do that!

Reply:
How did you do last time to propose changes?

Reply:
Didn't have time to get to it. I will work on it this week.

Reply:
@MaxUlysse:
![image](https://user-images.githubusercontent.com/465550/60503829-bd97b200-9cc0-11e9-8e1e-a32b011d6e40.png)

Reply:
Thanks a lot, I did not notice this icon

Reply:
```suggestion
```bash
docker-compose up --force-recreate -d
```

Reply:
```suggestion
docker-compose up --force-recreate -d
```